### PR TITLE
Content tweaks for building website (part of issue 2418 task 11)

### PR DESCRIPTION
**Pull request by alflennik**
The PR includes the practical miscellaneous changes needed to use the new content organization as the starting point to build the APG website. Just one example of the type of changes needed is the fact that the first sentence of each pattern is used as its description and the checkbox pattern did not actually begin with a complete sentence. The broken links were found as part of the link checker effort which itself should have a PR soon.

See related PR https://github.com/w3c/wai-aria-practices/pull/170 for more information.
___
WAI Preview Link [failed to build](https://github.com/w3c/wai-aria-practices/runs/9427753187?check_suite_focus=true) on 'Update site files' step. _(Last tried on Fri, 11 Nov 2022 03:39:17 GMT)._

### CODE REVIEW

**Comment by alflennik** [comment]
The build failure appears to be caused by [a caching issue in netlify](https://answers.netlify.com/t/error-fetching-branch-after-trying-to-update-submodule/5370/2), I will investigate this with Howard and get in contact with Steve at WAI who administers the ARIA-Practices Netlify account.

**Comment by mcking65** [comment]
@alflennik, I will review this afternoon (pacific time). I will try to do as early as possible so you have the feedbac/merge before end of your day.

When reaching out to Steve, also reach out to @shawna-slh. I believe @michael-n-cooper  told me that Steve was leaving at some point. So, there is a possibility he is not managing the netlify account. I might not be remembering correctly, and I don't recall timing. Just want to be sure you are able to get timely response.

**Review by mcking65 — APPROVED** [review]
@alflennik, Looks awesome! I made necessary editorial changes. I have a few questions:

1. Curious why img directory is in repo root instead of in say `/content/shared/`? Seems like we should consider images as part of the content.
2. Should the spec link fixing work if I view files locally in browser? It is not working for me.
3. Should insertion of example warning be working if I view example files locally? It is not working for me.
4. What do you think of renaming /content/index.html to /content/apg-home.html? I suggest this because we have another file named index.html under /content/index.
5. In /content/index.html, Could you please delete the `<div id="work"` that starts on line 61. We are not using it, and we don't have plans to restore it. It is most likely similar information will eventually end up under /about, and there is no reason to keep it in the home page file.
Feel free to merge this to move-examples branch whenever you are ready.

**Comment by shawna-slh** [comment]
Unfortunately, dependency updates broke many Netlify previews. Hopefully we can get those fixed soon.

File: content/shared/js/app.js [hunk 1]
```diff
@@ -6,63 +6,41 @@
...
+    const scriptSource = document
       .querySelector('[src$="app.js"]')
       .getAttribute('src');
-    // Replace 'app.js' part with 'notice.html'
+    const fetchSource = scriptSource.replace(
+      '/js/app.js',
+      '/templates/example-usage-warning.html'
+    );
+
+    // Load and parse the usage warning and insert it before the h1
```
[the remark this answers is not in the document]
Comment (alflennik) [reply content/shared/js/app.js:28]: Hi @mcking65, due to CORS, a local server must be running for both the example usage warning to be loaded and for the spec links to be rewritten. This is an existing limitation that also affects the usage warning on the `main` branch. If the URL begins with `file://`, browsers will not allow fetch or dynamic import to be used.

If you are on VSCode, I've found the `peakchen90.open-html-in-browser` extension provides an efficient development experience. With a html file open in the editor I can use a keyboard shortcut and the file will open in my default browser with a simple development server automatically running.

[the remark this answers is not in the document]
Comment (mcking65) [reply content/shared/js/app.js:28]: Thanks for the education and tip!!

**Comment by alflennik** [comment]
@mcking65 The feedback has been addressed, but before I merge I want to give you a chance to take a second pass over [the changes I've made since last time](https://github.com/w3c/aria-practices/pull/2537/files/e41a28fb4333d60b896dcd5de61b74ad23cb9d51..ac1e84b2f5931b6d8aec62e61c4355dbbe2800b1) which include fixing some garbled text and making the advisements consistent with notes and warnings.

File: content/shared/css/core.css [hunk 2]
```diff
@@ -129,7 +129,52 @@ th .example-header {
...
+
+.note > h2,
+.note > h3,
+.note > h4,
+.warning > h2,
+.warning > h3,
+.warning > h4,
+.advisement > h2,
+.advisement > h3,
+.advisement > h4 {
```
Comment (mcking65) [inline content/shared/css/core.css:158]: Should we consider specifying h5 even though we don't have any ... yet? There are a couple of practices, like keyboard, where we get down to h4 for subtopics.

**Review by mcking65 — APPROVED** [review]
Thank you @alflennik! Many excellent corrections. Great content quality boost! Just one forward looking question below in the CSS.

**Pull request merged**